lib/pages/chat/chat_view.dart

```diff
@@ -310,6 +285,31 @@ class ChatView extends StatelessWidget {
     );
   }
 
+  Widget inputMessageWidget(double bottomSheetPadding) {
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        ...[
+          const ConnectionStatusHeader(),
+          // Currently we can't support reactions
+          // ReactionsPicker(controller),
+          ReplyDisplay(controller),
+          ChatInputRow(controller),
+        ].map(
+          (widget) => Padding(
+            padding: EdgeInsets.only(
```

Comment:
https://api.flutter.dev/flutter/widgets/Wrap-class.html
should use Wrap and spacing

Reply:
This is old logic, I don't want to focus to refactor.